Sources/UIScreen+render.swift

```diff
@@ -91,14 +91,34 @@ extension UIScreen {
     }
 
     func outline(_ rect: CGRect, lineColor: UIColor, lineThickness: CGFloat) {
+        // we want to render the outline 'inside' the rect rather
+        // than exceeding the bounds when lineThickness is bigger than 1
+        let offset = lineThickness / 2
+        let scaledGpuRect = CGRect(
+            x: rect.origin.x + offset,
+            y: rect.origin.y + offset,
+            width: rect.size.width - offset,
+            height: rect.size.height - offset
+        ).gpuRect(scale: scale)
+
         GPU_SetLineThickness(Float(lineThickness))
-        GPU_Rectangle(rawPointer, rect.gpuRect(scale: scale), color: lineColor.sdlColor)
+        GPU_Rectangle(rawPointer, scaledGpuRect, color: lineColor.sdlColor)
     }
 
     func outline(_ rect: CGRect, lineColor: UIColor, lineThickness: CGFloat, cornerRadius: CGFloat) {
         if cornerRadius > 1 {
+            // we want to render the outline 'inside' the rect rather
+            // than exceeding the bounds when lineThickness is bigger than 1
+            let offset = lineThickness / 2
+            let scaledGpuRect = CGRect(
+                x: rect.origin.x + offset,
+                y: rect.origin.y + offset,
+                width: rect.size.width - offset,
+                height: rect.size.height - offset
+            ).gpuRect(scale: scale)
```

Comment:
nit:
Just wondering - since this logic is repeated twice - if it would be possible to have it in one place and somehow reuse it?

Reply:
i tried, but the readability really sucked after putting this into a helper function.

Reply:
No problem :) thanks for trying